### ISSUE TITLE
Crashes when opening

### ISSUE THREAD

**Issue by LilacZer0** [tahoma2d/tahoma2d#663]
It started yesterday when I wanted to start animating it just chrshed and I cant do nothing anymore what do I do?

**Comment by manongjohn** [comment]
Which version are you on? Are you getting any particular errors?

**Comment by LilacZer0** [comment]
> Which version are you on? Are you getting any particular errors?

I'm on 1.1 and idk if it is an error it just opens everything is white and when I click somthing it closes

**Comment by manongjohn** [comment]
Can you provide a screenshot or short video to show what is happening?

**Comment by LilacZer0** [comment]
how do i put a screenshot?

**Comment by LilacZer0** [comment]
#
![crash](https://user-images.githubusercontent.com/82464445/114588574-af1f4e00-9c3b-11eb-970f-53b2c36453c4.png)
here u go this is all I see when opening it

**Comment by manongjohn** [comment]
Odd...

Can you download a new version of Tahoma2D 1.1, put it in a different location and run it to see if it still works. I wonder if something corrupted your install? Either the crash did something strange or some Windows or Anti-virus update is messing up Tahoma suddenly.

**Comment by LilacZer0** [comment]
i did try that and it still did the same thing sadly I want to get this fixed cus I was having fun and this happens right when I make a schedual

**Comment by manongjohn** [comment]
Try downloading the 1.2 Alpha 2...see if that works. Really not sure what is going here...

**Comment by LilacZer0** [comment]
ok

**Comment by LilacZer0** [comment]
its still showing the white screen

**Comment by manongjohn** [comment]
I think something might have changed on your machine if its affecting both brand new downloads, but no idea what it could be.

If you are running any anti virus software, turn it off temporarily to see if that is somehow affecting it.

Do you know what kind of graphics card you have?

**Comment by LilacZer0** [comment]
i use NVIDIA GeForce GTX 1650 SUPER

**Comment by LilacZer0** [comment]
turning off the anti-virus did nothing

**Comment by artisteacher** [comment]
You might want to try checking the Nvidia site to see if there is an update for your driver.

**Comment by LilacZer0** [comment]
ok

**Comment by artisteacher** [comment]
This looks like the same thing as #660.

**Comment by manongjohn** [comment]
> This looks like the same thing as #660.

Yeah, I started wondering the same thing. Need to figure out what is causing that.

**Comment by manongjohn** [comment]
@LilacZer0 Just in case, try opening a command prompt with admin privileges and do a `sfc /scannow` in case something is messed up with your system. It may fix it. I've had Windows delete my bluetooth drivers once while I was trying to turn it on and this managed to fix it.

**Comment by LilacZer0** [comment]
nothing and im checking my driver for updates

**Comment by LilacZer0** [comment]
updating the drivers did nothing

**Comment by artisteacher** [comment]
If it's a driver issue, and it was working before, going back to an older driver could help.

**Comment by manongjohn** [comment]
@LilacZer0 I'd be curious to know OTX gives you the same problem. I just recently published OTX 1.5 which you can get [here](https://github.com/manongjohn/OTX/releases) or try one of the older 1.4 ones.  It's a portable just like Tahoma.

**Comment by LilacZer0** [comment]
OTX? is that opentoonz?

**Comment by manongjohn** [comment]
OTX is a portable version of OpenToonz which may have a few additional features not in the official release.

**Comment by LilacZer0** [comment]
i installed it and it works great

**Comment by LilacZer0** [comment]
sadly my other animation project won't load in but now I can try to make it even better

**Comment by manongjohn** [comment]
Weird. So OTX works but Tahoma doesn't....makes this even more of a mystery.

Something you can try doing for Tahoma...delete the cache folder:  C:\User\yourlogin\AppData\Local\Tahoma2D\Tahoma2D\cache

**Comment by LilacZer0** [comment]
nope still unuseable

**Comment by LilacZer0** [comment]
wait i think i found out the prob

when opentoonz opened for the first time it was fine on my main monitor and closed it on my drawing pad and now i open it again and now it's doing the same thing as Tahoma when it opens on my drawing tablet i don't think its the software i think its my tablet

**Comment by LilacZer0** [comment]
my tablet is making it crash somehow

**Comment by manongjohn** [comment]
What tablet do you have? Maybe try reinstalling the drivers for it?

**Comment by LilacZer0** [comment]
xp pen artis 12

**Comment by LilacZer0** [comment]
nope still nothing

**Comment by LilacZer0** [comment]
maybe the artist 12 is not compatible with it?

**Comment by LilacZer0** [comment]
krita works fine is it only those 2 software that don't work well with the artist 12?

**Comment by manongjohn** [comment]
I'm not sure why it wouldn't work on an artist 12.  I have an Artist 15.6 myself.  I can't imagine the drivers are that significantly different.

You said you were fine with 1.1 until it crashed.  How long had you been using 1.1 before that?

Some stuff you can try:
In Tahoma, try deleting the room layouts before opening it: Tahoma2d\tahomastuff\profile\users\yourlogin\layouts
If that doesn't work try deleting:  Tahoma2d\tahomastuff\profile\users\yourlogin\mainwindow.ini
If those don't work try duplicating your main screen to the tablet instead of extending it or make the tablet the main screen.

**Comment by LilacZer0** [comment]
1. nope
2. nope
3. HAZAA ITS WORKING NOW

**Comment by LilacZer0** [comment]
i open it and close it and no crashing its all good now thx for the help guys 👍

**Comment by manongjohn** [comment]
Well, at least we found a solution...may not be ideal but you can still keep working. :)

This kind of points to something might have changed with the display drivers, I think. Usually we recommend the duplicate/main screen switch if you have the drawing offset issue caused by different screen resolutions.  Might fix some other issues people are running into with display issues as well.

**Comment by LilacZer0** [comment]
im back and i have found the TRUE solution the problem was that both my main monitor and my tablet were not the same resolution. If they are both not the same resolution then Tahoma2d will crash and be unresponsive on startup on the graphic tablet. so both the main monitor and the graphic tablet would have to be set to 1920x1080 then Tahoma will be able to be used on the tablet with no issue

**Comment by artisteacher** [comment]
Thanks for the update! There is a known issue #783 with some dual monitor setups where the screens are multiple resolutions. In my case, opening T2D in low resolution fixed the issue. I wasn't getting any crashing on my end though.

**Comment by manongjohn** [comment]
Interesting. I've never heard of it crashing because of resolution differences either.
